pkgs/jdt-language-server/default.nix

```diff
@@ -74,8 +78,9 @@ stdenv.mkDerivation rec {
       #
       # Java options, such as -Xms and Xmx can be specified by setting JAVA_OPTS.
       makeWrapper ${jdk}/bin/java $out/bin/jdt-language-server \
-        --run "mkdir -p ${runtimePath}" \
-        --run "install -Dm 1777 -t ${runtimePath}/config $out/share/config/*" \
+        --run "mkdir -p ${configDirPath}" \
+        --run "mkdir -p ${dataDirPath}" \
+        --run "install -Dm 1777 -t ${configDirPath} $out/share/config/*" \
```

Comment:
Will the subshell paths be quoted? Does it matter?

Reply:
No they won't. It think it's okay because currently dir names can't have spaces in them.